src/src/CustomBuild/ESPEasyDefaults.h

```diff
@@ -320,12 +320,18 @@
 #endif
 
 #ifndef GITHUB_RELEASES_LINK_PREFIX
-# define GITHUB_RELEASES_LINK_PREFIX "<a href='https://github.com/letscontrolit/ESPEasy/releases' title='Click to show latest release on Github' target='_blank' style='font-size: 15px; text-decoration: none'>"
+# define GITHUB_RELEASES_LINK_PREFIX "<a href='https://github.com/letscontrolit/ESPEasy/releases' title='Click to show latest release on Github' target='_blank'>"
 #endif
 #ifndef GITHUB_RELEASES_LINK_SUFFIX
 # define GITHUB_RELEASES_LINK_SUFFIX "</a>"
 #endif
 
+#ifndef CDN_JS_CSS_URL
+#define CDN_JS_CSS_URL           "https://cdn.jsdelivr.net/gh/letscontrolit/ESPEasy@mega-20210503/static/"
```

Comment:
While we're at it, you might want to set it to at least the latest (current) build.

Reply:
I've just copied the embeded URL from the C++ function to the define...
Anything I've missed ?

Reply:
That URL may need an update anyway, since it is referring to an older tag (see URL)
Since it needs to be updated at a new build anyway, you may already set it to the currently last build.

I think I will have a look at the compile-time generated code to patch that URL on every tag/release builds
Just have to think about it, to make sure it will not break self made builds.

Reply:
It means that you automatically upload css&js to 
cdn.jsdelivr.net/gh/letscontrolit/ESPEasy@**TAG**/static/
for every release, right?

So what TAG should I put there ? the latest 20211105 ? or I simply let you add your automatic TAG, at build/release time, the way you decide to implement it?

Reply:
For now just use the latest one, as there is no new tag yet.
But I don't know how far back these older revisions will be available on the CDN servers.
I don't upload it, Github uses this as CDN platform for Github projects.

Reply:
I changed the tag in commit dbfd549, and added a reminder TODO comment, to update the tag at built time.

I hoppe that we are now able to merge.. ☺️